src/helpers/github.ts

```diff
@@ -152,9 +147,14 @@ export async function getOpenPullRequests(context: Context, targets: ReposWatchS
       return acc;
     }, []),
   ];
+  if (!monitor.length) {
+    filter.push(`org:${context.payload.repository.owner?.login}`);
```

Comment:
this is not part of the PR but I was just thinking that its possible a repo is not owned by an org, so actually it should check the owner type and default to org or repo

Reply:
@whilefoo It seems you can put a user name as an org and it works still:
https://github.com/search?q=org%3Awhilefoo+is%3Apr